Runtime/Managers/EditionMediator.cs

```diff
@@ -171,6 +181,28 @@ private void DeleteSelectedObjects(string stringIds)
 
         }
 
+        private void ChangeObjectsColor(string payload)
+        {
+            ChangeColorObjectMessagePayload parsedPayload = JsonUtility.FromJson<ChangeColorObjectMessagePayload>(payload);
+            List<GameObject> objectsToChangeColor = _changeColorManager.GetObjectsToChangeColor(parsedPayload.objectIds);
+            if (objectsToChangeColor.Count > 0)
+            {
+                Color? parsedColor = Utils.ParseColor(parsedPayload.color);
+                if (parsedColor != null)
+                {
+                    _changeColorManager.ChangeObjectsColor(objectsToChangeColor, (Color)parsedColor);
+                }
+                else
+                {
+                    SendErrorMessage("The color isn't valid");
```

Comment:
This would make a tiny bit easier
```suggestion
                    SendErrorMessage($"The color {parsedPayload.color} isn't valid");
```